.gitignore

```diff
@@ -23,5 +23,5 @@ neardev/dev-account
 # ignore out folder generated by build process
 out/
 
-# parcel config file
-.env.local
+# ignore IDE folders
+.idea
```

Comment:
Rather than putting all system / workspace-specific files like this in individual project `.gitignore` files, how would you feel about removing all such files (including `.DS_Store`) and having a policy of keeping such files in [your own global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead?

Reply:
We can't expect users who clone the repo to have them ignored globally.

Reply:
alright

Reply:
@willemneal does that matter? It's their project, at that point. I think giving them a `.gitignore` cluttered with a bunch of environmental concerns from various NEAR developers' machines is misleading; someone may think that this `.idea` file is part of a typical NEAR project. I'd rather have a few newbie developers end up accidentally committing `.DS_Store` and `.idea` files to their own repository (which is common), rather than potentially confuse them with a noisy starting `.gitignore`

Reply:
Yeah fair enough.